### ISSUE TITLE
Accessibility - (remaining) Characters not visible for screen readers (SC 4.1.3)

### ISSUE THREAD

**Issue by LaurensBurger** [open-formulieren/open-forms#5460]
### Thema / Theme

Frontend

### Omschrijving / Description

<img width="680" height="328" alt="Image" src="https://github.com/user-attachments/assets/0d1190cf-5b82-4dfd-be3d-83cc21da1262" />

suggested to use `aria-live="polite"` for these elements to inform the user on the changes.

**Comment by robinmolen** [comment]
This is a bit more complicated: see comment https://github.com/open-formulieren/formio-renderer/pull/123#discussion_r2235193879
